experimental/dds/tree2/src/core/undo/undoRedoManager.ts

```diff
@@ -173,20 +156,16 @@ export class UndoRedoManager<TChange, TEditor extends ChangeFamilyEditor> {
 
 	/**
 	 * Creates a copy of this `UndoRedoManager`.
-	 * @param getHead - Optional function to use for retrieving the head commit for the branch associated with the
-	 * the new {@link UndoRedoManager}. If one is not provided, the one from this {@link UndoRedoManager} will be used.
-	 * @param repairDataStoreProvider - Optional {@link IRepairDataStoreProvider} to use for the new {@link UndoRedoManager}.
-	 * If one is not provided, the `repairDataStoreProvider` of this {@link UndoRedoManager} will be cloned.
 	 * @param headUndoableCommit - Optional head undoable commit, if one is not provided the head undoable commit
 	 * of this {@link UndoRedoManager} will be used.
+	 * @param headUredoableCommit - Optional head uredoable commit, if one is not provided the head redoable commit
```

Comment:
Seems like as good a time as any to fix this:
```suggestion
	 * @param headRedoableCommit - Optional head redoable commit, if one is not provided the head redoable commit
```